calcom/cal.com
# Fixing navigation and visual issues

## What does this PR do?

Fixing navigation and visual issues with Admin Wizard app list as explained in #6066.

| Before      | After |
| ----------- | ----------- |
| ![image](https://user-images.githubusercontent.com/467258/208168952-47c0cc00-95cf-455b-bf69-1aeceb682fe4.png) | ![image](https://user-images.githubusercontent.com/467258/208168994-943d5330-a123-4ca9-a527-df332aabafe0.png) |

Fixes #6066

**Environment**: Staging(main branch) / Production

## Type of change


## How should this be tested?

Go to `/auth/setup` as an admin user and see app list correctly displayed and navegable.
